controllers/topics/inviteMembers.js

```diff
@@ -39,6 +57,36 @@ module.exports = async (req, res) => {
           return res.status(400).json(response);
         }
 
+        const invitations_msg = `${inviter.username} invited you to join ${topics.name} community`;
+
+        //topic invitation message primary chat
+        const topic_invitations_id = await uuid();
+        await TopicInvitations.create({
+          topic_invitations_id: topic_invitations_id,
+          user_id_inviter: req?.userId,
+          user_id_invited: user_id,
+          topic_id
+        });
+
+        const client = new StreamChat(process.env.API_KEY, process.env.SECRET);
+
+        const channel = await client.channel(
+          CHANNEL_TYPE_STRING.TOPIC_INVITATION,
+          topic_invitations_id,
+          {
+            name: `#${topics.name}`,
+            created_by_id: req?.userId,
+            members: [user_id],
+            channel_type: 5,
+            channel_image: topics.icon_path,
+            channelImage: topics.icon_path,
+            image: topics.icon_path
+          }
+        );
+        await channel.create();
+        await channel.sendMessage({user_id: req?.userId, text: invitations_msg}, {skip_push: true});
+        //end topic invitation message primary chat
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This section of the code seems to be handling the creation of topic invitations and sending messages via StreamChat. However, it lacks error handling for the operations that could potentially fail, such as creating a channel or sending a message. It would be beneficial to add try-catch blocks around these operations to handle any errors that may occur and respond appropriately.
```diff
-        const client = new StreamChat(process.env.API_KEY, process.env.SECRET);
-
-        const channel = await client.channel(
-          CHANNEL_TYPE_STRING.TOPIC_INVITATION,
-          topic_invitations_id,
-          {
-            name: `#${topics.name}`,
-            created_by_id: req?.userId,
-            members: [user_id],
-            channel_type: 5,
-            channel_image: topics.icon_path,
-            channelImage: topics.icon_path,
-            image: topics.icon_path
-          }
-        );
-        await channel.create();
-        await channel.sendMessage({user_id: req?.userId, text: invitations_msg}, {skip_push: true});
+        try {
+          const client = new StreamChat(process.env.API_KEY, process.env.SECRET);
+
+          const channel = await client.channel(
+            CHANNEL_TYPE_STRING.TOPIC_INVITATION,
+            topic_invitations_id,
+            {
+              name: `#${topics.name}`,
+              created_by_id: req?.userId,
+              members: [user_id],
+              channel_type: 5,
+              channel_image: topics.icon_path,
+              channelImage: topics.icon_path,
+              image: topics.icon_path
+            }
+          );
+          await channel.create();
+          await channel.sendMessage({user_id: req?.userId, text: invitations_msg}, {skip_push: true});
+        } catch (error) {
+          console.error(`Error creating channel or sending message: ${error}`);
+          // Handle error appropriately
+        }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->